recipes/pywinpty/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pywinpty" %}
+{% set version = "0.1.0" %}
+{% set hash = "106b9fcf793474626c61bb792e8ce1d850dc383081059f81bbf3028ff8abcb4c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/spyder-ide/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  skip: True  # [osx or linux or py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - cython
+    - python
+    - pywin32
+    - setuptools
+    - winpty
+  run:
+    - python
+    - pywin32
+    - winpty
+
+test:
+  imports:
+    - winpty
+
+about:
+  home: https://github.com/spyder-ide/pywinpty
+  license: MIT
```

Comment:
Please raise an issue upstream about getting the license file added to the `MANIFEST.in` so we can package it in the future.